frontend/web/components/ProjectManageWidget.tsx

```diff
@@ -0,0 +1,250 @@
+          environmentId: string
+          projectId: string
+        }) => {
+          router.history.push(
+            `/project/${projectId}/environment/${environmentId}/features?new=true`,
+          )
+        }}
+      />,
+      'p-0 side-modal',
+    )
+  }, [router.history])
+
+  useEffect(() => {
+    AppActions.getOrganisation(AccountStore.getOrganisation().id)
+  }, [])
+
+  useEffect(() => {
+    const { state } = router.route.location as { state: { create?: boolean } }
+    if (state && state.create) {
+      handleCreateProjectClick()
+    }
+  }, [handleCreateProjectClick, router.route.location])
+
+  return (
+    <div data-test='project-manage-widget' id='project-manage-widget'>
+      <OrganisationProvider>
+        {({ isLoading, projects }: OrganisationContext) => (
+          <div>
+            {(projects && projects.length) || isLoading ? (
+              <div className='flex-row pl-0 pr-0'></div>
+            ) : isAdmin ? (
+              <div className='container-mw-700 mb-4'>
+                <h5 className='mb-2'>
+                  Great! Now you can create your first project.
+                </h5>
+                <p className='fs-small lh-sm mb-0'>
+                  When you create a project we'll also generate a{' '}
+                  <strong>development</strong> and <strong>production</strong>{' '}
+                  environment for you.
+                </p>
+                <p className='fs-small lh-sm mb-0'>
+                  You can create features for your project, then enable and
+                  configure them per environment.
+                </p>
+              </div>
+            ) : (
+              <div className='container-mw-700 mb-4'>
+                <p className='fs-small lh-sm mb-0'>
+                  You do not have access to any projects within this
+                  Organisation. If this is unexpected please contact a member of
+                  the Project who has Administrator privileges. Users can be
+                  added to Projects from the Project settings menu.
+                </p>
+              </div>
+            )}
+            {(isLoading || !projects) && (
+              <div className='centered-container'>
+                <Loader />
+              </div>
+            )}
+            {!isLoading && (
+              <div>
+                <FormGroup>
+                  <PanelSearch
+                    id='projects-list'
+                    className='no-pad panel-projects'
+                    listClassName='row mt-n2 gy-4'
+                    title='Your projects'
+                    items={projects}
+                    renderRow={(
+                      { environments, id, name }: Project,
+                      i: number,
+                    ) => {
+                      return (
+                        <>
+                          {i === 0 && (
+                            <div className='col-md-6 col-xl-3'>
+                              <Permission
```

Comment:
Since this a functional component now, can we replace the <Permission components with the equivelent useHasPermission